### PR TITLE
user filter added

### PR DESCRIPTION
**Pull request by saurabh-singh-17**

### CODE REVIEW

**Comment by maheshkumargangula** [comment]
@saurabh-singh-17 - Please mention the JIRA ticket as part of the title.

File: self-service-portal/src/components/common/ParticipantInfo.tsx [hunk 1]
```diff
@@ -20,7 +20,9 @@ const ParticipantInfo = () => {
   const authToken = useSelector((state: RootState) => state.tokenReducer.participantToken);
   console.log("part details in dash", participantDetails, authToken);
 
+  useEffect(()=> {
   if(authToken == "abcd"){
+    console.log("came in abcd",)
```
Comment (maheshkumargangula) [inline self-service-portal/src/components/common/ParticipantInfo.tsx:25]: What is this logic?

File: self-service-portal/src/api/UserService.js [hunk 2]
```diff
@@ -28,9 +28,14 @@ export const serachUser = async (email) => {
     return post("/user/search", payload);
 }
 
-export const getAllUser = async () => {
+export const getAllUser = async (token="") => {
     var payload = { "filters": {} };
-    return post("/user/search", payload);
+    if(token != ""){
+        console.log("not blank ", token);
```
Comment (maheshkumargangula) [inline self-service-portal/src/api/UserService.js:34]: avoid logging the user token. At least use a debug logger.

**Review by maheshkumargangula — CHANGES_REQUESTED** [review]

File: self-service-portal/src/components/common/ParticipantInfo.tsx (hunk 1 above)
Comment (saurabh-singh-17) [reply self-service-portal/src/components/common/ParticipantInfo.tsx:25]: This is to see if actual token is generated and stored or not .. by default it is abcd to differentiate between actual token and dummy token

File: self-service-portal/src/api/UserService.js (hunk 2 above)
Comment (saurabh-singh-17) [reply self-service-portal/src/api/UserService.js:34]: debugging is not enabled in the app. ll take care of it later

**Review by maheshkumargangula — APPROVED** [review]

**Pull request merged**